packages/cra-template/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "cra-template",
-  "version": "1.0.3",
+  "version": "1.0.4",
+  "private": true,
```

Comment:
It's interesting that they have template as separate packages now - perhaps this would have been a better approach to create a backpack specific template instead of modifying the built in one.

Reply:
Yeah we have done this under here https://github.com/Skyscanner/cra-template-backpack - the cra/template in this repo is untouched

Reply:
Nice